src/pages/index.js

```diff
@@ -90,18 +92,71 @@ const DescriptiveText = styled(P)`
     color: ${colorMap.greyLightTwo};
     margin-bottom: ${size.medium};
 `;
+
+const ThumbnailCard = styled(Card)`
+    position: relative;
+`;
+
+const ThumbnailButton = styled(Button)`
+    left: 40%;
+    position: absolute;
+    top: 35%;
+`;
+
+const Thumbnail = ({ video }) => {
+    let thumbnailUrl = buildImage; //fallback image if there is none
+    let dimensionsMatcher = '{width}x{height}';
+    if (video.thumbnail_url) {
+        const containsSpace = video.thumbnail_url.match('%{width}x%{height}');
+        dimensionsMatcher = containsSpace
+            ? '%{width}x%{height}'
+            : dimensionsMatcher;
+        thumbnailUrl = video.thumbnail_url.replace(
+            dimensionsMatcher,
+            `${MEDIA_WIDTH}x${MEDIA_WIDTH}`
```

Comment:
Do we also need to add the `%`s here after replacing if it contains space?

Reply:
Nope! Those `%` cause the images to error out and should not be there. So this logic not only sets the dimensions but removes those `%`'s as well.

Reply:
I wonder how these thumbnails get saved, because these thumbnails are being returned with the `%`'s in there.